llvm/docs/LangRef.rst

```diff
@@ -1161,15 +1161,17 @@ parameters of a function. Parameter attributes are considered to be part
 of the function, not of the function type, so functions with different
 parameter attributes can have the same function type.
 
-Parameter attributes are simple keywords that follow the type specified.
-If multiple parameter attributes are needed, they are space separated.
-For example:
+Parameter attributes are simple keywords that follow the specified type.
+When multiple parameter attributes are required, they are separated by
+spaces. Additionally, target-dependent attributes can be provided as a
```

Comment:
I don't think the terminology "target-dependent attributes" is really in use anymore, these are simply "string attributes". We have plenty of target-independent string attributes as well.

Reply:
Thanks, Nikita. I have updated the changes.